.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
```

Comment:
**note:**
This can be easily changed to automated nightly build by adding a schedule instead if that's what you'd prefer. I've kept the tag -> release approach, as that's what I do at work most of the time :)